pkg/clients/grpc/grpc.go

```diff
@@ -239,52 +215,3 @@ func ReadAttestationPolicy(manifestPath string, attestationConfiguration *check.
 
 	return ErrAttestationPolicyMissing
 }
-
-func CustomDialer(ctx context.Context, addr string) (net.Conn, error) {
```

Comment:
Is this not needed anymore, i.e. it was not necessary in the first place?

Reply:
we need the custom dialer since we add tls extensions and standard go library does not allow for this